starport/services/serve/conf/config.go

```diff
@@ -32,12 +47,26 @@ type Validator struct {
 	Staked string `yaml:"staked"`
 }
 
+// Servers keeps configuration related to started servers.
+type Servers struct {
```

Comment:
I added server address configurability in this PR, in order to be able to spin up multiple apps by running `serve`  multiple times so ports won't conflict witch each other.